packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -124,12 +124,12 @@ export class RpcContractProvider
    *
    * @see https://tezos.gitlab.io/api/rpc.html#get-block-id-context-big-maps-big-map-id-script-expr
    */
-  async getBigMapKeyByID<T>(
+  async getBigMapKeyByID(
```

Comment:
removing <T> seems to be a breaking change for the users. I don't think we should make that change.